.github/workflows/update_github_io.yml

```diff
@@ -18,15 +18,16 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          repository: stoyanK7/checkstyle
```

Comment:
Not good

Reply:
I know, running actions right now on local to make sure everything is good to go.

Reply:
Done. Should be good to go.